src/Model/Block/YouTube.php

```diff
@@ -2,21 +2,32 @@
 
 namespace eLife\ApiSdk\Model\Block;
 
+use eLife\ApiSdk\Collection\Sequence;
 use eLife\ApiSdk\Model\Block;
+use eLife\ApiSdk\Model\BlockWithCaption;
 use eLife\ApiSdk\Model\HasId;
 
-final class YouTube implements Block, HasId
+final class YouTube implements BlockWithCaption, HasId
 {
```

Comment:
`HasId` is overloaded (as it was before), as it may refer to an HTML id or to a YouTube video id; luckily they are both strings but one has stronger validations than the other.

Reply:
See: https://github.com/elifesciences/api-raml/pull/225#discussion_r204667059